src/components/container.js

```diff
@@ -0,0 +1,41 @@
+import Input from "./input"
+import List from "./todo_item"
+import useLocalStorage from "./localstorage";
+import { Container as ChakraContainer } from '@chakra-ui/react'
+
+function Container(){
+    const [value,setValue] = useLocalStorage("todos")
+    function Submitted(data){
```

Comment:
It's not recommended to use Pascal case as function because it's recognized by React as a component